docs/get_started/Installation.md

```
@@ -11,6 +11,16 @@
 
 We currently provide three methods to install **TileScale**:
 
+**(optional)Prepare the container**:
+
+```bash
+docker pull nvcr.io/nvidia/pytorch:25.03-py3
+docker run --name tilescale --ipc=host --network=host --privileged --cap-add=SYS_ADMIN --shm-size=10g --gpus=all -it nvcr.io/nvidia/pytorch:25.03-py3 /bin/bash	
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 160

(MD010, no-hard-tabs)